ProcessLib/HydroMechanics/FluidType.cpp

```diff
@@ -0,0 +1,17 @@
+#include "FluidType.h"
+
+namespace FluidType
+{
+
+FluidType strToFluidType(std::string s)
```

Comment:
I think it is better to use a `const reference` for passing the parameter.

```suggestion
FluidType strToFluidType(std::string const& s)
```